instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
Also, currently I don't think we use persistent DBs on any of our sandboxes, no? Since you mentioned SWIF is only activated when persitent databases are, the impact should be minimal in any case?